utils/src/main/kotlin/Extensions.kt

```diff
@@ -233,6 +243,11 @@ fun JsonNode?.fieldsOrEmpty(): Iterator<Map.Entry<String, JsonNode>> = this?.fie
  */
 fun JsonNode?.textValueOrEmpty(): String = this?.textValue().orEmpty()
 
+/**
+ * Converts this [Long] from bytes to mebibytes (MiB).
+ */
+fun Long.bytesToMiB(): Double = this / 1048576.0
```

Comment:
It seems a bit arbitrary to only have this for `Long`. How about using `Number` instead? And could we do the IMO more readable `(1024 * 1024)` instead of `1048576.0`?

Reply:
Using `Number` does not allow to use the `/` operator, it also makes sense to have it `Long` because that is the type returned by `File.length()`, so file sizes are usually of this type. Changed it to use `(1024.0 * 1024.0)`.

Reply:
> Using Number does not allow to use the `/` operator

I guess that could have been easily solved by using `toDouble()` instead of `this`. That way you could also write `(1024 * 1024)` instead of `(1024.0 * 1024.0)` as the number already is a double.